src/Uno.UI/UI/Xaml/Controls/FocusVisual/SystemFocusVisual.wasm.cs

```diff
@@ -0,0 +1,39 @@
+﻿#nullable enable
+
+using Uno.Foundation;
+using Uno.UI.Xaml.Core;
+using Uno.UI.Xaml.Input;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Input;
+
+namespace Uno.UI.Xaml.Controls;
+
+internal partial class SystemFocusVisual : Control
+{
+	private const string JsType = "Windows.UI.Xaml.Input.FocusVisual";
+
+	partial void AttachVisualPartial()
+	{
+		if (FocusedElement == null)
+		{
+			return;
+		}
+
+		WebAssemblyRuntime.InvokeJS($"{JsType}.attachVisual({HtmlId}, {FocusedElement.HtmlId})");
```

Comment:
How often is this invoked ?

Reply:
Only once when focused element changes (e.g. once per tab key press)

Reply:
Is it invoked when the UI is built, e.g. when multiple items are added in sequence? I'm wondering if we should be doing a TSInterop entry instead.

Reply:
I don't think so, focus should be set only once per page load, but will check

Reply:
I verified and this will be called only when the keyboard focused element actually changes - which will happen only on keyboard presses or when a new page is loaded - but even in that case, it will happen only when the first focusable element is set.